whois-api/src/test/java/net/ripe/db/whois/api/ElasticSearchHelper.java

```diff
@@ -70,7 +65,7 @@ public void resetElasticIndexes(final String indexName, final String metaDetaInd
         }
     }
 
-    @NotNull
+    @Nonnull
```

Comment:
why this change?
Is NotNull wrong used? why we prioritise @Nonnull

Reply:
We are being inconsistent in using *two* types of annotation :
* JSR 305 javax.annotation.Nonnull
* Jetbrains IDEA import org.jetbrains.annotations.NotNull
I prefer to use *one* annotation (Nonnull) everywhere

Reply:
We could (maybe should) migrate away from Nonnull e.g. see 
https://github.com/google/guava/issues/2960
But that's a separate issue